src/Aspire.Hosting.Azure/Provisioning/Provisioners/BicepProvisioner.cs

```diff
@@ -94,7 +94,7 @@ await notificationService.PublishUpdateAsync(resource, state =>
 
             return state with
             {
-                State = new("Running", KnownResourceStateStyles.Success),
+                State = new("Provisioned", KnownResourceStateStyles.Success),
```

Comment:
I see the following states while a ServiceBus is being provisioned:

<img width="635" alt="image" src="https://github.com/user-attachments/assets/ea2807dc-5c1d-4fad-8f6f-a024f236a29e" />

Seems weird that the top-level resource is provisioned but the underlying resources are pending and starting?